packages/devtools/src/launcher.js

```diff
@@ -120,6 +120,8 @@ function launchBrowser (capabilities, product) {
 
     if (!executablePath) {
         throw new Error('Couldn\'t find executable for browser')
+    } else if (!executablePath.toLowerCase().includes(CHANNEL_FIREFOX_NIGHTLY)) {
```

Comment:
This check should be skipped if we don't want to start Firefox.

Reply:
Haa! Yes. I just missed that Edge also uses the same function. My bad! Will fix this. Thanks!